test/specs/contextpopup/ContextPopupDirective.spec.js

```diff
@@ -16,6 +16,12 @@ describe('ga_contextpopup_directive', function() {
       $provide.value('gaNetworkStatus', {
         offline: true
       });
+
+      $provide.value('gaLang', {
+        get: function() {
+          return 'de';
+        }
+      });
```

Comment:
We load the entire ga module for each test. So the topic request is launched asynchronously.